scripts/prep_carlsson_pdbs.py

```diff
@@ -0,0 +1,56 @@
+import sys, os, argparse, yaml
+
+sys.path.append(
+    f"{os.path.dirname(os.path.dirname(os.path.abspath(__file__)))}"
+)
```

Comment:
We shouldn't need to do this, what are you trying to accomplish with this?

Reply:
added issue here: https://github.com/choderalab/covid-moonshot-ml/issues/48